src/App.tsx

```diff
@@ -1,10 +1,57 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { Notification } from './components/Notification';
+import { StatusTodos } from './types/StatusTodo';
 
-const USER_ID = 0;
+const USER_ID = 9946;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState('all');
```

Comment:
You have an enum for the `all` value
```suggestion
  const [status, setStatus] = useState(StatusTodos.ALL);
```